packages/apps/dev-wallet/src/pages/create-profile/create-profile.css.ts

```diff
@@ -0,0 +1,52 @@
+import { atoms, tokens } from '@kadena/react-ui/styles';
+import { style } from '@vanilla-extract/css';
+
+export const authCard = style([
+  atoms({
+    padding: 'xxl',
+  }),
+  {
+    borderRadius: '1px',
+    textAlign: 'left',
+    backgroundColor: 'rgba(255, 255, 255, 0.03)',
+    border: '1px solid rgba(255, 255, 255, 0.1)',
+  },
+]);
+
+export const inputClass = style([
```

Comment:
not sure if we should override styling of the input as if it will be re-designed/re-branded on the react-ui it will not be updated here